### DISCUSSION TITLE
Add "Your Position" to Daily Challenge page on web

### DISCUSSION THREAD

**Discussion by rileywaffle** [ppy/osu#29265] [Ideas]
I think having something like this that denotes your position on the current daily challenge, like something similar to this attached photo 

![iJ5FnJeWOK](https://github.com/user-attachments/assets/22fa3d4a-4786-4d5f-a6a4-fd95d6c68471)

https://osu.ppy.sh/multiplayer/rooms/811018?page=1#scores on pages like this to be specific.

**Comment by ReinIsNOTaDev** [comment]
Daily Challenge probably should be moved to a different page on web, and then would it probably be implements

**Comment by peppy** [comment]
cc @nanaya
